meson.build

```diff
@@ -15,32 +16,82 @@ if not meson.is_subproject()
     werror = true
 endif
 
+script = join_paths(meson.source_root(), 'workaround.sh')
```

Comment:
This seems terribly wrong. meson 0.55.0 added a `patch_directory` option to handle locally wrapped dependencies, so you do not need to autogenerate tarball intermediaries here.

Reply:
@eli-schwartz The yocto build we're using (and can't change for a while) is using meson 0.53.  We've not been able to iron out a set of LD based issues for the cross compiler calls with meson, which is why CMake got added.  I've not been able to get back to trying to find a better solution for this, but good suggestion.

@guruchandru I'd rather not add the workaround code to the meson file since that couples us with a hack that hopefully we can remove sooner than later.

Reply:
In that case, you cannot rely on the `[provide]` section either, so you can just add the `fallback: ['foo_project', 'foo_dep']` styled references to each one, and you still do not need if/else logic. :)

Reply:
As far as these wraps go... jansson could probably go into the wrapdb, the other ones look like they may be specialized enough that that is not worth it so the alternative would be somehow getting meson.build files into those repos...